modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/viewmodel/PodcastViewModel.kt

```diff
@@ -210,7 +212,10 @@ class PodcastViewModel
     }
 
     fun updatePodcast(existingPodcast: Podcast) {
-        podcastManager.refreshPodcastInBackground(existingPodcast, playbackManager)
+        // Refresh the podcast application coroutine scope so the podcast continues to update if the view model is closed
+        applicationScope.launch {
+            podcastManager.refreshPodcast(existingPodcast, playbackManager)
+        }
```

Comment:
I thought about moving this to a work manager task but as ideally, the podcast would be refreshed instantly I have left it as a Coroutine launch.